### DISCUSSION TITLE
Where should customized wav file be put?

### DISCUSSION THREAD

**Discussion by hua-bing** [gtxaspec/wz_mini_hacks#795] [Q&A]
I just installed it on a V2.  I am trying to get some audio files onto the SD card using scp.  But the copy always failed due to the target (/media/mmc/wz_mini/usr/share/audio/) being a read-only file system.  Apparently, somebody has done the same [successfully](https://github.com/gtxaspec/wz_mini_hacks/issues/108#issuecomment-1159613242).  Could anybody pls help?  Thanks.

**Comment by HeyItsDaddy** [comment]
I'm using V3's, and that's where I copied my files using WinSCP without any problems.  That location shouldn't be read-only.  If you're still having problems you could always just pull the SD card and copy the files locally and then put the card back in, though.  Nice as nice as doing it remotely, but it gets the job done.

**Reply by hua-bing** [reply]
Thanks.  I've been pulling the SD card out and copying.  I am playing with a V2.  The "hacking" processes are different btw V2 and V3.  Probably that makes the difference.  Looks I have to live with it.
